index.html

```diff
@@ -31,23 +30,27 @@
                 <a href="#about">About</a>
                 <a href="#formato">Contact</a>
             </div>
+           
```

Comment:
- [ ] [OPTIONAL] The desktop nav links seem to be bigger than they should be. They shouldn't overlap each other. I suggest setting the font-size to the required size as seen in the figma design.

![image](https://user-images.githubusercontent.com/4247740/161857412-82cccfc7-2cc7-4fb1-bb91-981d53d25b7a.png)

Reply:
ohhh didnt check the last changes of desktop thank you very much